repository-data/webfiles/src/main/resources/site/freemarker/common/hero.ftl

```diff
@@ -26,9 +25,9 @@
 <#assign hasColourBar = isTall?then(displayColourBar, false) />
 
 <#if hasDocument>
-    <#assign isCtaDoc = document.class.simpleName?starts_with("Calltoaction") />
-    <#assign isBannerDoc = document.class.simpleName?starts_with("Banner") />
-    <#assign isVideoDoc = document.class.simpleName?starts_with("Video") />
+    <#assign isCtaDoc = document.class.simpleName == "Calltoaction"/>
+    <#assign isBannerDoc = document.class.simpleName == "Banner"/>
+    <#assign isVideoDoc = document.class.simpleName == "Video"/>
```

Comment:
On another ticket I had to change `document.class.simpleName == "Video"` to `document.class.simpleName?starts_with("Video")` to fix a bug on the main site where videos stopped shoping on UAT. 

On the main site the simple name is something like `Video@Video...` hence `starts_with`. Please explore this change further.